src/main/java/org/zalando/nakadi/repository/kafka/KafkaPartitionStartStatistics.java

```diff
@@ -0,0 +1,49 @@
+package org.zalando.nakadi.repository.kafka;
+
+import org.zalando.nakadi.domain.NakadiCursor;
+import org.zalando.nakadi.domain.PartitionBaseStatisticsImpl;
+import org.zalando.nakadi.domain.PartitionStartStatistics;
+import org.zalando.nakadi.domain.Timeline;
+
+public class KafkaPartitionStartStatistics extends PartitionBaseStatisticsImpl implements PartitionStartStatistics {
+
+    private final KafkaCursor first;
+
+    public KafkaPartitionStartStatistics(final Timeline timeline, final int partition, final long firstOffset) {
+        super(timeline, KafkaCursor.toNakadiPartition(partition));
+        this.first = new KafkaCursor(timeline.getTopic(), partition, firstOffset);
+    }
+
+    @Override
+    public NakadiCursor getFirst() {
+        // TODO: Support several timelines
+        return first.toNakadiCursor(getTimeline());
+    }
+
+    @Override
+    public NakadiCursor getBeforeFirst() {
+        // TODO: Support several timelines
+        return first.addOffset(-1).toNakadiCursor(getTimeline());
```

Comment:
What happens if `first` is `0`? I think that, in that case, we should get the last offset of the previous non-empty timeline, correct?